.github/workflows/build_gradle_sam.yml

```
@@ -0,0 +1,26 @@
+name: di-gradle-sam
+
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - 'aws-sam/*'
+      - 'gradle-sam/*'
```

```suggestion
      - 'gradle-sam/*'
      - '.github/workflows/build_gradle_sam.yml'
```
So that changing the workflow triggers a rebuild

---

added this and removed `aws-sam` as that'll be captured by the suggestion below